packages/lesmis/src/components/DataGrid/DataGrid.js

```diff
@@ -1,178 +0,0 @@
-/*
- * Tupaia
- * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
- *
- */
-import React from 'react';
-import PropTypes from 'prop-types';
-import styled from 'styled-components';
-import { useTable, useFilters, useSortBy, usePagination } from 'react-table';
-import { Table, TableHead, TableRow, TableBody, TableSortLabel } from '@material-ui/core';
-import { TableCell, StyledTableRow } from '@tupaia/ui-components';
```

Comment:
Was there anything left in `ui-components` that becomes unused across all packages after this?